Framework/src/CheckRunner.cxx

```diff
@@ -314,18 +319,18 @@ QualityObjectsType CheckRunner::check()
                       << ENDM;
 
   QualityObjectsType allQOs;
-  for (auto& check : mChecks) {
-    if (updatePolicyManager.isReady(check.getName())) {
-      auto newQOs = check.check(mMonitorObjects);
+  for (auto& tuple : mChecks) {
```

Comment:
```suggestion
  for (auto& [checkName, check] : mChecks) {
```
something like this would be more clear, same comment above

Reply:
totally agree